frontend/src/components/Cam.vue

```diff
@@ -0,0 +1,183 @@
+
+<template>
+<div ref="mask" class="modal-mask">
+    <div class="modal-wrapper">
+    <div class="modal-container">
+
+        <div class="modal-header">
+        <slot name="header">
+            <h3>Say Cheese!</h3>
+        </slot>
+        </div>
+
+        <div class="modal-body">
+        <slot name="body"></slot>
+        <div ref="box" class="video-box">
+            <canvas ref="canvas" id="canvas" width="640" height="480"></canvas>
+            <video ref="video" id="video" width="640" height="480" autoplay></video>
+            <div class="video-controls" ref="button">
+                <div class="video-snap">
+                    <button id="snap" class="btn-snap" v-on:click="capture()">
+                        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><circle cx="12" cy="12" r="3.2"/><path d="M9 2L7.17 4H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2h-3.17L15 2H9zm3 15c-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5-2.24 5-5 5z"/><path d="M0 0h24v24H0z" fill="none"/></svg>
```

Comment:
SVG los trekken en in een aparte file en directory zetten. Deze vervolgens importeren in deze file.